k4e-agent-sti.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+set -e
+
+# The scripts performs the following steps on rhel-8.5:
+# * Install imagebuilder 
+# * Create rpms for yggdrasil and k4e-device-worker
+# * Publish rpms as yum repo to be consumed by imagebuilder
+# * Publish aarch64 rhel-image for edge-device
+
+# The script assumes the machine is registered via subscription-manager
+if ! subscription-manager refresh; then
+   echo  "The machine is not registered."
+   exit 1
+fi
+
+#---------------------------
+# Install osbuild components
+#---------------------------
+dnf install -y osbuild-composer composer-cli cockpit-composer bash-completion
+systemctl enable osbuild-composer.socket --now
+systemctl enable cockpit.socket --now
+systemctl start firewalld
+firewall-cmd --add-service=cockpit && firewall-cmd --add-service=cockpit --permanent
```

Comment:
In setup script we don't touch firewalld. Is there a different scenario here?

Reply:
i don't think the cockpit itself is needed here, since the script uses only the cli.
but that's part of the official documentation for configuring the image-builder

Reply:
I used rhel with all its defaults. I changed nothing and it worked